Kudu.Core/Functions/FunctionManager.cs

```diff
@@ -49,52 +51,214 @@ public FunctionManager(IEnvironment environment, ITraceFactory traceFactory)
         public bool IsFunctionEnabled()
         {
             // this should read appSettings instead
-            var hostJson = Path.Combine(_environment.WebRootPath, HostJsonFile);
+            var hostJson = Path.Combine(_environment.WebRootPath, Constants.FunctionsHostConfigFile);
```

Comment:
`WebRootPath` should be `FunctionsPath`? You have the same Path.Combine 3 times, so maybe a helper.